cli/exchange/org.go

```diff
@@ -0,0 +1,165 @@
+package exchange
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/open-horizon/anax/cli/cliutils"
+	"github.com/open-horizon/anax/i18n"
+	"net/http"
+)
+
+type ExchangeOrgs struct {
+	Orgs      map[string]interface{} `json:"orgs"`
+	ListIndex int                    `json:"lastIndex"`
+}
+
+type OrgHeartbeat struct {
```

Comment:
Please use the HeartbeatIntervals struct from the exchange package instead of defining your own.

Reply:
Done. Removed my struct and replaced it with exchange.HeartbeatIntervals